test_logwarn

```diff
@@ -99,6 +99,7 @@ is-ignored '\[warn\].* fatal: Invalid revision range .*\.\.'
 is-ignored '\[warn\] \[pid:[0-9]*\] $'
 is-ignored '\[error\].*cmd returned 32768$'
 is-ignored '\[error\] Worker [0-9]* has no heartbeat .[0-9]* seconds., restarting'
+is-ignored '\[error\] Could not render a response at'
```

Comment:
the original line has `[error] [j0WyMARuTaBr]`, so I would do
```suggestion
is-ignored '\[error\] .*Could not render a response at'
```

Reply:
Right, at this place it is apparently the "generic pattern" again. It feels kind of redundant having to specify the generic pattern in the code *and* the test.